Shepherd/reduction

FIREdog5 opened the pull request:
core shepherd code

Closes #610

FIREdog5 committed:
rm -r ;)

FIREdog5 committed:
cleaned out Shepherd.py

FIREdog5 committed:
cleaned out Shepherd.py again :)

FIREdog5 committed:
cut down utils

kylesu98 committed:
deleted timer types

ziyaointl committed:
clean up Alliance.py

ziyaointl committed:
add todo in Alliance.py

FIREdog5 committed:
removed previous web pages

FIREdog5 committed:
Merge branch 'shepherd/reduction' of https://github.com/pioneers/PieCentral into shepherd/reduction

FIREdog5 committed:
deleted more (maybe done?)

ziyaointl committed:
Update .gitignore

Add rule for vscode

ziyaointl committed:
Clean up Sensors.py

Deleted goal and bid related code. They should be useful as a reference though.

ziyaointl committed:
Add Pipfile

ziyaointl committed:
Comment out unused class header in Utils.py

ziyaointl committed:
Clean up dawn_server.py

jialinli98 committed:
sensor

FIREdog5 committed:
disabled import error

FIREdog5 committed:
un-disabled import error

charleshong3 commented on shepherd/Pipfile:
```diff
@@ -0,0 +1,20 @@
+
+[requires]
+
+python_version = "3.6"
```
This file should be deleted

charleshong3 commented on shepherd/Pipfile.lock:
```diff
@@ -0,0 +1,132 @@
+        }
+    },
+    "develop": {}
+}
```
delete this too

ziyaointl committed:
Remove duplicate pipfiles

Apparently Jonathan added a more update-to-date version in #625, so I'm deleting this one

jonathan-j-lee reviewed (approved):
LGTM.

ziyaointl committed:
Merge branch 'master' into shepherd/reduction
